third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -859,6 +865,7 @@ impl Interpreter {
                 )
             })
             .collect();
+        println!("{:?}", stack_trace);
```

Comment:
Should this `println!` be added to `get_internal_state()` instead of here?

Reply:
Sry I need to get rid of this...